### PR TITLE
Creating a User model

### PR DESCRIPTION
**Pull request by Lu-Tola07**

### CODE REVIEW

File: model/userModel.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    email: {
+        type: String,
+        // unique: true
+    },
+    passWord: {
+        type: String
+    }
+});
+
```
Comment (folakeatolagbe96) [inline model/userModel.js:26]: add timestamp

**Review by folakeatolagbe96 — APPROVED** [review]

**Review by Ahmeddavids — APPROVED** [review]

File: model/userModel.js
Comment (Miraclepio) [inline model/userModel.js:1]: why is email not unique sir?

**Review by Miraclepio — APPROVED** [review]

**Review by Sylvergold — COMMENTED** [review]
Add timestamps

**Review by OtegaClinton — APPROVED** [review]

File: model/userModel.js
Comment (Lu-Tola07) [reply model/userModel.js:1]: Would work on it sir.

**Review by Cjoseph8 — COMMENTED** [review]
just tesing it

**Pull request merged**